docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -1,4 +1,4 @@
-- name: C# Compiler Errors
+- name: C# compiler messages
```

Comment:
This toc node contains both errors and warnings. I updated it to C# compiler messages.